### DISCUSSION TITLE
How to create multiple pages and make a button to go into them?

### DISCUSSION THREAD

**Discussion by zSynctic** [flet-dev/flet#723] [Q&A]
for example I have a sign up button when clicked I want it to take them to another page from the username and password how can I create another page and change between them

**Comment by Francois-Belanger** [comment]
https://flet.dev/blog/navigation-and-routing/

**Comment by saurabhwadekar** [comment]
You can try using flet-route it will be easy for https://github.com/saurabhwadekar/flet_route
